src/global/config.ts

```diff
@@ -0,0 +1,63 @@
+import { DEFAULT_GLOBAL_CONFIG } from './constants';
+
+type ConfigData = {
```

Comment:
Might be worthwhile to interlace the keys with `DEFAULT_GLOBAL_CONFIG` for IDE autocompletion but we can revisit that in the future.

Reply:
Nice catch, I managed to break this one too in the late-night refactor